src/language-css/index.js

```diff
@@ -54,7 +54,7 @@ const languages = [
 
 const postcss = {
   get parse() {
-    return eval("require")("./parser-postcss");
+    return require("./parser-postcss");
```

Comment:
My assumption in removing these evals is that they were used to stop browserify or webpack from bundling them as dependencies. Since the approach of this PR is to load 0 plugins from the web route, it should avoid that requirement entirely.

Reply:
I think the evals are there mainly to stop ... rollup? ... from bundling those files in the npm package. Remember, Prettier bundles all its dependencies into the npm package and has zero dependencies listed in package.json. I might remember all of this wrong though.